File: FitProj/app/src/main/java/com/teuskim/fitproj/FitApplication.java
package com.teuskim.fitproj;

import android.app.Application;

import com.teuskim.fitproj.common.FitDao;

/**
 * 앱의 어플리케이션 클래스
 */
public class FitApplication extends Application {

    @Override
    public void onTerminate() {
        FitDao.getInstance(getApplicationContext()).close();
        super.onTerminate();
    }
}

File: FitProj_kotlin/app/src/main/java/com/teuskim/fitproj/FitApplication.kt
package com.teuskim.fitproj

import android.app.Application

import com.teuskim.fitproj.common.FitDao

/**
 * 앱의 어플리케이션 클래스
 */
class FitApplication : Application() {

    override fun onTerminate() {
        FitDao.getInstance(applicationContext)!!.close()
        super.onTerminate()
    }
}
